ArcadiaParties-SPA/src/app/department/department-selector/department-selector.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component } from '@angular/core';
+import { Store, select } from '@ngrx/store';
+import { Router } from '@angular/router';
+import { Observable, combineLatest } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { DepartmentsState } from '../reducers/reducer';
+import { loadDepartmentsAction } from '../actions/actions';
+import { selectDepartments, selectCurrentDepartment } from '../selector/selector';
+import { User } from 'src/app/user/models/User';
+import { selectUser } from 'src/app/user/selector/selector';
+import { Department } from '../models/Department';
+
+@Component({
+  selector: 'app-department-identity',
```

Comment:
It is not `identity`.

Reply:
Class and component selector have been renamed